src/data/bucket/circle_bucket.js

```diff
@@ -81,4 +81,6 @@ class CircleBucket extends Bucket {
     }
 }
 
+CircleBucket.programInterface = circleInterface;
```

Comment:
I'm confused -- is this related to the architectural documentation or something else?